packages/block-library/src/tag-cloud/index.php

```diff
@@ -51,5 +51,14 @@ function register_block_core_tag_cloud() {
 			'render_callback' => 'render_block_core_tag_cloud',
 		)
 	);
+
+	register_block_style(
```

Comment:
Any particular reason why this style isn't included in the `block.json` file? See: https://github.com/WordPress/gutenberg/blob/trunk/docs/reference-guides/block-api/block-metadata.md#block-styles.